src/builds/appveyor.rs

```diff
@@ -0,0 +1,90 @@
+use std::io::Read;
+
+use chrono::{DateTime, UTC};
+use diesel;
+use diesel::prelude::*;
+use hyper::Client;
+use hyper::header::Accept;
+use hyper::net::HttpsConnector;
+use hyper_native_tls::NativeTlsClient;
+use serde::Deserialize;
+use serde_json;
+
+use DB_POOL;
+use domain::builds::Build;
+use error::DashResult;
+
+static ROOT_URL: &'static str = "http://ci.appveyor.com/api";
+
+pub fn get_build(build: &str) -> DashResult<()> {
```

Comment:
might want something a little more descriptive like `get_and_insert_build` to make it clear we've got side effects (maybe the same for travis?)